week-2/13-order/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Flexbox</title>
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/week2.css">
+	<link rel="stylesheet" href="/css/week2-flexbox.css">
+  <link rel="stylesheet" href="flexbox.css">
+  <link rel="stylesheet" href="./ignore_this_folder/custom.css">
+</head>
+
+<body>
+  <div class="site-wrapper site-wrapper">
+    <div class="site-header">
+      <h1 class="site-header__title">First</h1>
+    </div>
+    <ul class="countries countries--first">
+			<li class="country" id="brazil">Brazil</li>
```

Comment:
I've been using 2-space tab indentation. Do you know what other modules are using? We should probably standardize everything on beautify or prettier.

Reply:
Yes, I'll add a PR after this with "prettier" as a recommended VS extensions (so it recommends installing it when somebody opens the project) and then reformat all html and css files with prettier